ui/components/app/step-progress-bar/index.scss

```diff
@@ -104,22 +107,38 @@ ul.two-steps {
 }
 
 .progressbar li.two-steps::after {
-  width: 180px;
   height: 2px;
   content: '';
   position: absolute;
   background-color: var(--color-background-alternative);
   top: 15px;
-  right: 75px;
+  right: 45%;
   z-index: -1;
 
+  @media screen and (max-width: 349px) {
+    right: 39%;
+  }
+
+  @media screen and (min-width: 350px) and (max-width: 460px) {
+    right: 42%;
+  }
+
+  @media screen and (min-width: 461px) and (max-width: $break-small) {
+    right: 44%;
+  }
+
+  @media screen and (min-width: $break-small) {
+    right: 46%;
+  }
+
+
```

Comment:
Can remove all these media queries and right positioning now 

```suggestion

```

Local with the above removed

<img width="1440" alt="Screen Shot 2022-04-23 at 4 04 36 PM" src="https://user-images.githubusercontent.com/8112138/164948919-f08141d8-d51c-44ee-8800-a2a3068d52b2.png">
<img width="1440" alt="Screen Shot 2022-04-23 at 4 04 45 PM" src="https://user-images.githubusercontent.com/8112138/164948920-0e0ccb23-b426-4118-b2d8-cf4ffdc9f26e.png">